### DISCUSSION TITLE
Netbox process continously exit with code 0

### DISCUSSION THREAD

**Discussion by arunkumarviswanathan** [netbox-community/netbox#16379] [Help Wanted!]
Hi,

I'm getting the following Netbox exit message and Netbox startup continuously in the Netbox container, and as per the log the NextBox process is getting killed and a new Netbox process is getting started. 
![image](https://github.com/netbox-community/netbox/assets/64256917/5b8cf8b9-79a4-47ef-8045-65c1cc3cc940)

Because of the above issue, the Netbox is unstable and throwing 503 error more frequently. We are not able to have a smooth UI or API experience. 
Also, the Error/Notice message is mentioned as an expected one during the Netbox startup in the troubleshooting guide (https://github.com/netbox-community/netbox-docker/wiki/Troubleshooting). Still, it is happening continuously in my setup and not allowing to use the Netbox properly. 

Any help or suggestions could help us to fix this error.

I appreciate any help you can provide.

**Comment by RakeshAMD** [comment]
same issue here any solution ?

**Comment by mkeuschn** [comment]
Looks like that there is a idle_timeout setting which is causing this behaviour. The nginx unit webserver is shutting down the application. If you do a request afterwards the app must be started again which is taking some time.

https://github.com/netbox-community/netbox-docker/blob/release/docker/nginx-unit.json
```
"applications": {
    "netbox": {
      "type": "python 3",
      "path": "/opt/netbox/netbox/",
      "module": "netbox.wsgi",
      "home": "/opt/netbox/venv",
      "processes": {
        "max": 4,
        "spare": 1,
        "idle_timeout": 120
      }
    }
```
